elixir/apps/domain/priv/repo/migrations/20231102042428_add_routing_to_gateway_group.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Domain.Repo.Migrations.AddRoutingToGatewayGroup do
+  use Ecto.Migration
+
+  @create_query "CREATE TYPE routing_type_enum AS ENUM ('managed', 'self_hosted', 'stun_only', 'turn_only')"
```

Comment:
So far we don't use any db enums because they are a pain to maintain (require DDLs and migrations), let's just use a string field and rely on in-app logic.

Reply:
Ok, was just trying to make sure there was no way to get bad data in to the DB, but will take this out.